src/irmin/node_intf.ml

```diff
@@ -212,8 +222,9 @@ module type Graph = sig
   (** [value_t] is the value type for {!value}. *)
 end
 
-module type Node = sig
+module type Sigs = sig
   module type S = S
+  module type Metadata = Metadata
```

Comment:
I don't think we need to expose it here, but as you want.

Reply:
I think that's useful to expose for users to define their own metadata modules (to use with `Make_ext`)